### DISCUSSION TITLE
LLDPD does not show neighbors of neighbors

### DISCUSSION THREAD

**Discussion by HectorFire** [lldpd/lldpd#513] [Q&A] [answered]
Hi Vicent,

Currenly I'm working in a team developing a device that they will be sharing its information using lldp. We are usig your source code and the proposal is to identify all devices in our network.
Currently when we use the command **lldpcli show neighbors** our devices only show the information of the neighbor directly conected, but we are thinking that it should shows all neighbor in the network.
Could you confirm that lldpd should display the information of neigbors of neighbors? if the answer if yes do you think that we are missing configure a parameter of lldpd?
The following image shows you our current topology and the results printed.

![image](https://user-images.githubusercontent.com/105220953/167423936-e97bb470-be53-446d-9f64-1ee088852fcb.png)

Thanks for your answer.
Hector Garrido.

**Comment by boomertsfx** [comment, marked as answer]
That would be a ton of information to exchange on a large network.  I prefer it to just show what device I'm directly connected to.  Our monitoring system uses SNMP LLDP to map out the network that way.

**Reply by vincentbernat** [reply]
Yes, that's correct. Use SNMP to gather all the neighbors. LLDP does not have provision to advertise neighbors of neighbors.
